agenta-backend/agenta_backend/models/api/user_models.py

```diff
@@ -12,7 +12,7 @@ class User(TimestampModel):
     uid: str
     username: str
     email: str  # switch to EmailStr when langchain support pydantic>=2.1
-    organization_id: str
+    organizations: Optional[List]
```

Comment:
List[str], no?

Reply:
organizations is a list of OrganizationDB and should be `Optional[List[Organization]]` but I can't do that because the presence of this line `from agenta_backend.models.api.user_models import User` in agenta-backend/agenta_backend/models/api/user_models.py would cause a circular import error if I try to import Organization from agenta-backend/agenta_backend/models/api/organization_models.py.

Reply:
> organizations is a list of OrganizationDB and should be `Optional[List[Organization]]` but I can't do that because the presence of this line `from agenta_backend.models.api.user_models import User` in agenta-backend/agenta_backend/models/api/user_models.py would cause a circular import error if I try to import Organization from agenta-backend/agenta_backend/models/api/organization_models.py.

@devgenix, he's actually right. You can even do it like so:

Optional[List[ObjectId]]

The data that's going to be stored there is a list of organization IDs, what better way to do it than to explicitly define the data type that'll be stored in the list.